test/unit/test_complex.py

```diff
@@ -114,7 +114,8 @@ def test_floordiv(self):
 
     def test_richcompare(self):
         self.assertIs(complex.__eq__(1+1j, 1 << 10000), False)
-        self.assertRaises(TypeError, complex.__lt__, 1+1j, None)
+        # in py3 this is returns NotImplemented so comment this out
```

Comment:
We're not in py3 in this file (although I imagine this change came across from somewhere where it's relevant!)

Reply:
yeah this was getting ahead of myself. I'll remove this test change until it makes sense to do so.